sdk/identity/identity/src/credentials/managedIdentityCredential/index.ts

```diff
@@ -117,6 +122,9 @@ export class ManagedIdentityCredential implements TokenCredential {
         `${ManagedIdentityCredential.name} - Client Id and Resource Id can't be provided at the same time.`,
       );
     }
+    if (_options?.retryOptions?.maxRetries !== undefined) {
```

Comment:
the only nit here is that we're not really allowing anyone to override the startDelayInMs but to be honest with MSAL MI around the corner we can make minimal changes here especially as it only pertains to 404s